Removing call information because is close now (#723)
ropensci/roweb3: content/champions/_index.md

The rOpenSci Champion Program [is part of a series of activities and projects we are carrying out to ensure our research software serves everyone in our communities](/blog/2021/12/20/inclusive-leadership-program/) and is supported through a grant from the Chan Zuckerberg Initiative.

## Program 2023-2024 timeline

> Our second cohort started this January! This is the tentavite timeline for 2024.

Reviewer: ```suggestion
Our second cohort started this January! This is the tentative timeline for 2024.
```

|Date|Event|Detail|Duration|
|----|-----|------|--------|
|January 2024|Kick-off event|Learn about the program, review the program expectations, and address questions. Meet mentors.|1.5 h|

> |February - April 2024 | Training|Knowledge of various channels through which new members can engage in and contribute to rOpenSci and R projects (by rOpenSci)|1 h|

Reviewer: ```suggestion
|February - April 2024 | Training|Knowledge of various channels through which new members can engage in and contribute to rOpenSci and R projects (by rOpenSci). |1 h|
```

> |February - April 2024| Training|Technical skills in software development (by rOpenSci)|2 h|

Reviewer: ```suggestion
|February - April 2024| Training|Technical skills in software development (by rOpenSci). |2 h|
```

> |February - April 2024 |Training| Technical skills in software review. (by rOpenSci)|2 h|

Reviewer: ```suggestion
|February - April 2024 |Training| Technical skills in software review (by rOpenSci). |2 h|
```

> |March-July 2024|Develop project. Meeting with mentors|Go through the review process as an author, or as a reviewer, or build the package of their choice||

Reviewer: ```suggestion
|March-July 2024|Develop project; Meet with mentor |Go through the review process as an author, or as a reviewer, or build a package. ||
```

> |May 2024|Cohort meeting: Presentation of projects| Champions' 5 minutes presentations of their project (package review or development) and get feedback.|1.5 h|

Reviewer: ```suggestion
|May 2024|Cohort meeting: Project Presentations| Champions give 5 min presentations of their projects (package review or development) and receive feedback.|1.5 h|
```

> |July 2024 |Cohort meeting: Reflections on projects|Champions will be invited to share takeaways from their projects with the cohort.|1.5 h|

Reviewer: ```suggestion
|July 2024 |Cohort meeting: Project Reflections|Champions will be invited to share takeaways from their projects with the cohort.|1.5 h|
```

> |August - September 2024 | Training|How to plan and facilitate engaging and inclusive workshops to support participants’ success (by CSCCE)|2 h|

Reviewer: ```suggestion
|August - September 2024 | Training|How to plan and facilitate engaging and inclusive workshops to support participants’ success (by CSCCE).|2 h|
```

> |August - October 2024|Champions outreach: host events, deliver talk, write about their experiences.|Champion's host: a workshop on good practices to build packages or good practices in software peer review or deliver a talk on a conference/event and one community event to gather feedback to improve rOpenSci’s projects|1 to 3 h|

Reviewer: Have I understood correctly? this is two options? Workshop or Give two talks?
```suggestion
|August - October 2024|Champions outreach: Host events, deliver talks, write about their experiences.|Champions host a workshop on good practices to build packages or good practices in software peer review; or deliver a talk at a conference/event and at one community event to gather feedback to improve rOpenSci’s projects. |1 to 3 h|
```

|September 2024|Cohort meeting: Reflections on events |Champions will be invited to share takeaways from their events with the cohort.|1.5 h|
|October 2024|Cohort meeting: Reflections on program|Champions will be invited to share takeaways from the program with the cohort and start to define future roles for Champions.|1.5 h|
|November 2024|End of term|Close cohort and define new roles for Champions and Mentors.|1 h|

## How to Apply ?

> The application are closed.  We will announce when we open a new call for champions and mentors. 

Reviewer: I thought to add a bit of cheer to the end. As in "you missed it, but you can see how things go!"

Although I realized that there are some `champions` tags in the [tag list](https://ropensci.org/tags/champions). Should those be `champions-program`?
```suggestion
2023-2024 applications are closed.  We will announce when we open a new call for champions and mentors. In the meanwhile, follow the progress of this cohort and the last through [rOpenSci Blog posts](/tags/champions-program/).
```

Author: Yes those should be champions-program
